conn/bind_std.go

```diff
@@ -6,64 +6,133 @@
 package conn
 
 import (
+	"context"
 	"errors"
 	"net"
 	"net/netip"
+	"strconv"
 	"sync"
 	"syscall"
+
+	"golang.org/x/net/ipv4"
+	"golang.org/x/net/ipv6"
+)
+
+var (
+	_ Bind = (*StdNetBind)(nil)
 )
 
 // StdNetBind is meant to be a temporary solution on platforms for which
 // the sticky socket / source caching behavior has not yet been implemented.
 // It uses the Go's net package to implement networking.
 // See LinuxSocketBind for a proper implementation on the Linux platform.
```

Comment:
Since this file now replaces bind_linux, I suppose this comment can be deleted/updated?